### ISSUE TITLE
doc error:torch.nn.NLLLoss

### ISSUE THREAD

**Issue by laizewei** [pytorch/pytorch#15163]
if size_average=True and reduce=True,
loss(x,class)=−weights[class]∗x[class]/N

**Comment by SsnL** [comment]
Please describe the issue in more details. Otherwise we can't understand what you mean.

**Comment by laizewei** [comment]
DOC:
if size_average=True and reduce=True,
loss(x,class)=−weights[class]∗x[class]/(∑n=1toN (Wyn))
∑n=1toN (Wyn) ≠N

**Comment by SsnL** [comment]
Use the issue template.
